componentsPage.html

```diff
@@ -215,7 +215,95 @@ <h2>Icon Buttons</h2>
+                </ol>
+                <ol class="list-style-lower-roman">
+                    <li>list Item1</li>
+                    <li>list Item2</li>
+                    <li>list Item3</li>
+                </ol>
+                <ol class="list-style-upper-roman">
+                    <li>list Item1</li>
+                    <li>list Item2</li>
+                    <li>list Item3</li>
+                </ol>
+            </div>
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28175%2C191%2C211%2C0.97%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--%2520This%2520is%2520default%2520list%2520--%253E%250A%2520%2520%253Col%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%2520%2520%253C%252Fol%253E%250A%2520%2520%253Col%2520class%253D%2522list-style-decimal-leading-zero%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%2520%2520%253C%252Fol%253E%250A%2520%2520%253Col%2520class%253D%2522list-style-lowerCase-alphabet%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%2520%2520%253C%252Fol%253E%250A%2520%2520%253Col%2520class%253D%2522list-style-upperCase-alphabet%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%2520%2520%253C%252Fol%253E%250A%2520%2520%253Col%2520class%253D%2522list-style-lower-roman%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%2520%2520%2520%253C%252Fol%253E%250A%2520%2520%2520%253Col%2520class%253D%2522list-style-upper-roman%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item1%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item2%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253Elist%2520Item3%253C%252Fli%253E%250A%2520%2520%2520%253C%252Fol%253E"
+            style="width: 538px; height: 763px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Rather than writing inline styling everywhere, it would be more manageable if you create a class and use it everywhere.